macros/EmbedInteractiveExample.ejs

```diff
@@ -0,0 +1,31 @@
+<%
+// Embeds a live sample from a mdn.github.io GitHub page
+//
+// Parameters:
+//  $0 - The URL of mdn.github.io page (relative)
+//  $1 - Keyword indicating iframe size: small, medium, or large
+//
+//  Example call {{EmbedInteractiveExample("interactive-examples/pages/css/animation.html", "small")}}
+//
+
+let url = "https://mdn.github.io/" + $0;
+let classesList = Array();
+
+// figure out classes
+if($0.includes('/js/')){
+    classesList.push('interactive-js');
```

Comment:
*face, palm*

Personally, I find 2 spaces as useful as 0. When scanning, the indent doesn't stand out to me at all. I wish tabs had won so we could all set our editors to show as much or as little indent as possible and didn't have to go from the tabs vs spaces argument to the 2 vs 4 spaces argument. But that ship has sailed :/